src/clj/athens/self_hosted/clients.clj

```diff
@@ -70,24 +71,26 @@
         valid-server-event?   (schema/valid-server-event? data)]
     (if (or valid-event-response?
             valid-server-event?)
-      (do
+      (let [type   (common-events/find-event-or-atomic-op-type data)
+            status (:event/status data)]
         (log/debug "Sending to username:" username
                    ", event-id:" (:event/id data)
-                   ", type:" (:event/type data)
-                   ", status:" (:event/status data))
+                   (if type
+                     (str ", type: " type)
+                     (str ", status: " status)))
         (http/send! channel (->transit data)))
       ;; TODO internal failure mode, collect in reporting
       (log/error "Not sending invalid event to username:" username
                  ", event-id:" (:event/id data)
-                 ", type:" (:event/type data)
+                 ", type:" (common-events/find-event-or-atomic-op-type data)
```

Comment:
`find-event-or-atomic-op-type` doesn't seem to have defined behaviour or tests for invalid events, and events that fall here are invalid in one way or another.